adapters/repos/db/helpers/helpers.go

```diff
@@ -25,6 +28,49 @@ var (
 	DocIDBucket                = []byte("doc_ids")
 )
 
+func MakePropertyPrefix(property string, propIds *tracker.JsonPropertyIdTracker) ([]byte, error) {
+	propid, err := propIds.GetIdForProperty(string(property))
+	if err != nil {
+		fmt.Printf("property '%s' not found in propIds, creating", property)
+	}
+	propid_bytes := make([]byte, 8)
+	binary.LittleEndian.PutUint64(propid_bytes, propid)
+	return propid_bytes, nil
+}
+
+func MakePropertyKey(propPrefix []byte, key []byte) []byte {
+	if len(propPrefix) == 0 {
+		return nil
+	}
+
+	t := key[:]
+	val := append(t, propPrefix...)
```

Comment:
why are you doing `t := key[:]` before appending to it?

Reply:
> true, but key is already a slice

The compiler disagrees

Reply:
mine does not complain. after removing the line db starts without any issues

Reply:
Try the test now

Reply:
it might be fixed on the latest version, but I doubt it

Reply:
The code in question was a bad cut-n-paste from some defensive code, but we do need a copy there